assets/stylesheets/_mixins.scss

```diff
@@ -333,3 +333,13 @@
 	// icon standards.
 	margin-right: 2px;
 }
+
+/**
+ * Allows users to opt-out of animations via OS-level preferences.
+ */
+
+@mixin reduce-motion {
+	@media (prefers-reduced-motion: reduce) {
+		animation-duration: 1ms !important;
```

Comment:
I know I'm late to this merged tickets party but thought I'd mention the lack of `transition-duration: 1ms !important;` in this mixin.

Not sure if a `transition` is even used anywhere but it would make the mixin a little more versatile.

Reply:
Good call — we could try that too. I wonder if it'd make sense to create two different mixins — one for `transition`, and one for `animation`. Just so that we aren't including of unused `!important` rules when we don't need them. I may be overthinking it though. 😄

In any case, seems worth exploring! Feel free to open up a PR.